test/include/seqan3/test/performance/naive_minimiser_hash.hpp

```diff
@@ -0,0 +1,121 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Mitra Darvish <mitra.darvish AT fu-berlin.de>
+ * \brief Provides seqan3::views::naive_minimiser_hash.
+ */
+
+#pragma once
+
+#include <range/v3/view/sliding.hpp>
+
+#include <seqan3/alphabet/concept.hpp>
+#include <seqan3/core/type_traits/range.hpp>
+#include <seqan3/range/views/detail.hpp>
+#include <seqan3/range/views/kmer_hash.hpp>
+#include <seqan3/range/views/minimiser.hpp>
+#include <seqan3/std/ranges>
```

Comment:
```suggestion
#include <seqan3/std/ranges>

#include <range/v3/view/sliding.hpp>

#include <seqan3/alphabet/concept.hpp>
#include <seqan3/core/type_traits/range.hpp>
#include <seqan3/range/views/detail.hpp>
#include <seqan3/range/views/kmer_hash.hpp>
#include <seqan3/range/views/minimiser.hpp>
```

`seqan3/std/ranges` is sorted as if it already is the `<ranges>` header